papers/ring_vrf/pedersen_vrf.tex

```diff
@@ -118,27 +121,31 @@ \section{Ring VRFs from the Pedersen VRF}
 \item $\rVRF.\rSign\eprint{ : }{}(\sk,\openring,\msg,\aux)\eprint{ \mapsto \rho}{}$
 	returns $\rho = (\compk,\piring,\sigma)$ \eprint{where \\}{\\ where} 
 	\tmpindent $(\openpk,\compk) \leftarrow \PedVRF.\CommitKey$,  \\
-	\tmpindent $\piring \leftarrow \NIZK_{\Lring}.\Prove(\compk,\comring,\openpk,\openring)$, \\
+	\tmpindent $\piring \leftarrow \NIZK_{\Lring}.\Prove((\openpk,\openring),(\compk,\comring))$, \\
 	\tmpindent $\aux' \leftarrow \tmpaux$,  \\
 	\tmpindent $\sigma \leftarrow \PedVRF.\Sign(\sk,\openpk,\msg, \aux')$.
 \item $\rVRF.\rVerify : (\comring,\msg,\aux,\rho) \mapsto \Out \,\, \lor \perp$ \,
-	parses $\rho$ as $(\compk,\piring,\sigma)$, \eprint{next }{}sets $\aux' \leftarrow \tmpaux$,
-	aborts if $\NIZK_{\Lring}.\Verify(\piring,\compk,\comring)$ fails,
+	parses $\rho$ as $(\compk, \comring,\piring,\sigma)$, \eprint{next }{}sets $\aux' \leftarrow \tmpaux$,
+	aborts if $\NIZK_{\Lring}.\Verify((\compk, \comring),\piring)$ fails,
 	and returns $\PedVRF.\Verify(\compk,\msg, \aux', \sigma)$.
 \end{itemize}
 
 
 % Although \PedVRF itself exhibits surprising properties, our gestalt 
-\rVRF satisfies sensible security definitions:
-Pseudo-randomness holds by reduction to singleton rings.
-Ring uniqueness, ring unforgeability, and ring anonymity resemble security
-arguments for other ring signatures built from SNARKs.
+
+%NOTE: The below paragraph is not formally stated and not related with our security proof
+%\rVRF satisfies sensible security definitions:
+%Pseudo-randomness holds by reduction to singleton rings.
+%Ring uniqueness, ring unforgeability, and ring anonymity resemble security
+%arguments for other ring signatures built from SNARKs.
+
```

Comment:
I took this out for now, but added TODOs for Oana here.  We'll see what happens.